deps/graph-parser/src/logseq/graph_parser/property.cljs

```diff
@@ -43,7 +43,7 @@
    #{:id :custom-id :background-color :background_color :heading :collapsed
      :created-at :updated-at :last-modified-at :created_at :last_modified_at
      :query-table :query-properties :query-sort-by :query-sort-desc :ls-type
-     :hl-type :hl-page :hl-stamp :file-path}
```

Comment:
@xyhp915 Was this removed so that users could edit the `file-path` property? If so, it should be added to `editable-built-in-properties` or else this creates an accidental page for a built-in property

Reply:
This is a special property.

>  `:file-path` was added to the hidden attribute in the last version, resulting in an "empty" block added to the top of the "hls___" page every time it is highlighted. Removing it from the list is a temporary fix, @xyhp915 will resolve this with a better UI.